src/app_service/service/community_tokens/service.nim

```diff
@@ -120,8 +115,7 @@ QtObject:
 
   proc getCommunityTokens*(self: Service, communityId: string): seq[CommunityTokenDto] =
     try:
-      let chainId = self.networkService.getNetworkForCollectibles().chainId
-      let response = tokens_backend.getCommunityTokens(communityId, chainId)
```

Comment:
Why is this being removed?

Reply:
Because we need the list of all deployed smart contracts, regardless chainId.